cmd/mev-boost/main.go

```diff
@@ -16,21 +19,28 @@ var (
 	// defaults
 	defaultPort         = 18550
 	defaultExecutionURL = getEnv("EXECUTION_URL", "http://127.0.0.1:18545")
-	defaultRelayURL     = getEnv("RELAY_URL", "http://127.0.0.1:28545")
+	defaultRelayURL     = getEnv("RELAY_URLS", "http://127.0.0.1:28545")
```

Comment:
To make it consistent ?
```suggestion
	defaultRelayURLs     = getEnv("RELAY_URLS", "http://127.0.0.1:28545")
```